uizacoresdk/build.gradle

```diff
@@ -18,7 +18,9 @@ android {
         //https://stackoverflow.com/questions/41098015/how-to-set-vectordrawable-as-an-image-for-imageview-programmatically
         //https://stackoverflow.com/questions/49014416/set-vector-drawable-to-imageview-in-xml
         //https://stackoverflow.com/questions/34417843/how-to-use-vector-drawables-in-android-api-lower-21
-        vectorDrawables.useSupportLibrary = true*/
+        */
+        vectorDrawables.useSupportLibrary = true
```

Comment:
Currently, the sdk still not use `vector`, So, should we enable this from application side?